src/supermarket/Gemfile

```diff
@@ -82,7 +82,7 @@ group :test do
 end
 
 group :development, :test do
-  gem "chefstyle", "2.1.3"
+  gem "chefstyle"
```

Comment:
why are we using this `cookstyle` gem in common gems as well as inside group?

Reply:
This is chefstyle which is used for our ruby code vs. cookstyle which is used for cookbook code